packages/core/tsconfig.json

```diff
@@ -1,12 +1,19 @@
 {
   "compilerOptions": {
+    "target": "es2022",
     "declaration": true,
-    "declarationDir": "./types",
+    "declarationDir": "./main",
```

Comment:
is this necessary? The package.json still points to `types` for typings, and I think it's a better practice.

Was this due to the next.js configuration?

Reply:
no. this is because i met some problems when publishing and with types. i experimentally changed the config try to solve the problem. and it was changed back to `"./main"` now. No worry.